src/style.css

```diff
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+}
```

Comment:
don't apply anything except box-sizing to *
better put font-family to `html `selector and inherit where needed

Reply:
When I do this, my tests do not pass. 1px is added to the indentation of the text and I have to eliminate it with crutches